src/lib.rs

```diff
@@ -99,5 +99,21 @@ macro_rules! assign_resources {
                 }
             }
         );
-    }
+    };
+    {
+        $(
+            $(#[$outer:meta])*
+            $group_name:ident : $group_struct:ident {
+                $(
+                    $(#[$inner:meta])*
+                    $resource_name:ident : $resource_field:path $(=$resource_alias:ident)?),*
```

Comment:
The only change to the signature is on this line, `ident->path`:
```suggestion
                    $resource_name:ident : $resource_field:<HERE> $(=$resource_alias:ident)?),*
```
Do not apply this suggestion, it is not a suggestion, merely using it to point out the exact change in the macro argument signature.